modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,127 @@ The properties for broker-wide throughput quota balancing are configured at the
+To set a throughput quota for a single client, use the `rpk cluster quotas alter` command. 
 
-The values of both `target_quota_byte_rate` and `target_fetch_quota_byte_rate` are throughput rate limits within a shard, in bytes per second.
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id-prefix=consumer-
+----
 
 === Group of clients throughput limit
 
-To set throughput quota for a group of clients, configure the cluster properties xref:reference:cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`] and xref:reference:cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`].
+Alternatively, you can view or configure throughput quotas for a group of clients based on a match on client ID prefix. The following example sets the `consumer_byte_rate` quota to a group of clients that have IDs starting with `consumer-`:
+
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id-prefix=consumer-
+----
+
+NOTE: The client group that matches on `client-id-prefix` does not denote a Kafka consumer group. Instead, it is the set of clients fetching from a Redpanda broker that are configured by the property to be throughput limited.
+
+=== Default client throughput limit
 
-* The `kafka_client_group_byte_rate_quota` property applies to producer clients. It sets a maximum throughput quota of traffic sent to Redpanda from each producer in the group named by the property.
-* The `kafka_client_group_fetch_byte_rate_quota` property applies to consumer clients. It sets a maximum throughput quota of traffic fetched from Redpanda by each consumer in the group named by the property.
+You can apply default throughput limits to clients. Redpanda applies the default limits if no quotas have been configured specific to a client ID or ID prefix. Defaults are only configurable based on the exact client ID.
 
-Both `kafka_client_group_byte_rate_quota` and `kafka_client_group_fetch_byte_rate_quota` have the configuration value fields:
+The example below sets a produce quota of 1GBps through the Kafka API, which means a 1GBps limit across all produce requests sent to a single Redpanda broker. 
 
-* `group_name`: a name for the group of clients.
-* `clients_prefix`: a client belonging to the group has this prefix in its `client_id` property.
-* `quota`: the maximum throughput rate of *each client in the group* in bytes per second.
+[,bash]
+----
+rpk cluster quotas alter --default client-id --add producer_byte_rate=1000000000
+----
 
-NOTE: For `kafka_client_group_fetch_byte_rate_quota`, a group of consumer clients is not a Kafka consumer group, and the `group_name` field is not a Kafka consumer group ID. Instead, it is the set of clients fetching from a Redpanda broker that are configured by the property to be throughput limited.
+[[migrate]]
+=== Migrate cluster configuration quotas to Kafka API-based quotas
```

Comment:
@pgellert Will ask Rogger to review rpk command syntax -- can you confirm if these are the correct migration steps? cc @mattschumpert

Reply:
Yes, these are the steps necessary. A couple of things that I think would be useful to add here:
* I think we should call out the per-shard vs per-broker distinction here again. If let's say they had `target_fetch_quota_byte_rate` set to 100MBps/shard and they are running on 16 core brokers, they might want to set the new quotas to 1600MBps.
* I think a table would be useful here to show what cluster configs correspond to what rpk arguments:
```
target_quota_byte_rate                      | --default client-id --add producer_byte_rate=<value>
target_fetch_quota_byte_rate                | --default client-id --add consumer_byte_rate=<value>
kafka_admin_topic_api_rate                  | --default client-id --add controller_mutation_rate=<value>
kafka_client_group_byte_rate_quota          | --name client-id-prefix=<prefix> --add producer_byte_rate=<value>
kafka_client_group_fetch_byte_rate_quota    | --name client-id-prefix=<prefix> --add consumer_byte_rate=<value>
```
* `rpk cluster config get kafka_client_group_byte_rate_quota` would not return a number (`1000000000` like in the example). It has a more complex structure. So I would either replace the example with `rpk cluster config get target_fetch_quota_byte_rate`. Or, even better, change it such that `kafka_client_group_byte_rate_quota` returns something that's expected, eg:
```
% rpk cluster config get kafka_client_group_byte_rate_quota
"kafka_client_group_byte_rate_quota": [
  {
    "group_name": "group_1",
    "clients_prefix": "producer_group_alone_producer",
    "quota": 10240
  },
  {
    "group_name": "group_2",
    "clients_prefix": "producer_group_multiple",
    "quota": 20480
  },
]
% rpk cluster quotas alter --add producer_byte_rate=10240 --name client-id-prefix=producer_group_alone_producer
% rpk cluster quotas alter --add producer_byte_rate=20480 --name client-id-prefix=producer_group_multiple
```